Ginger/GingerCoreNETUnitTest/RecordingLibTest/TestDriver.cs

```diff
@@ -9,6 +9,11 @@ public class TestDriver : IRecord
     {
         private bool LearnAdditionalDetails { get; set; }
         public event RecordingEventHandler RecordingEvent;
+        object Amdocs.Ginger.Plugin.Core.IRecord.CurrentBusinessFlow
```

Comment:
after removing CurrentBusinessFlow from IRecord, please remmber to remove it from here and all drivers

Reply:
Done